### PR TITLE
Changes to easily allow for ETH/USD and BTC/USD in spotPrice format

### PR DESCRIPTION
**Pull request by spuddyminer**
spotPrice feeds for ETH/USD and BTC/USD have been added, and the legacy feeds have been labeled as "legacy". 

These changes make it easier for reporters to submit ETH/USD and BTC/USD as SpotPrice query type.

After editing and creating new `eth_usd_feed.py` and `btc_usd_feed.py` files, I searched through `telliot-feeds` to make sure that python can still import the feeds properly. I also reported the feeds on rinkeby in both legacy and SpotPrice format.

### CODE REVIEW

File: src/telliot_feeds/reporters/interval.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 from telliot_feeds.datafeed import DataFeed
 from telliot_feeds.feeds import CATALOG_FEEDS
-from telliot_feeds.feeds.eth_usd_feed import eth_usd_median_feed
+from telliot_feeds.feeds.eth_usd_legacy_feed import eth_usd_legacy_feed
```
Comment (oraclown) [inline src/telliot_feeds/reporters/interval.py:24]: Use the spot price feed versions here

File: src/telliot_feeds/reporters/custom_reporter.py [hunk 2]
```diff
@@ -14,7 +14,7 @@
 from web3.datastructures import AttributeDict
 
 from telliot_feeds.datafeed import DataFeed
-from telliot_feeds.feeds.eth_usd_feed import eth_usd_median_feed
+from telliot_feeds.feeds.eth_usd_legacy_feed import eth_usd_legacy_feed
```
Comment (oraclown) [inline src/telliot_feeds/reporters/custom_reporter.py:17]: Use spot price feeds here rather than legacy

File: tests/feeds/test_eth_usd_feed.py [hunk 3]
```diff
@@ -2,20 +2,20 @@
 
 import pytest
 
-from telliot_feeds.feeds.eth_usd_feed import eth_usd_median_feed
+from telliot_feeds.feeds.eth_usd_legacy_feed import eth_usd_legacy_feed
```
Comment (oraclown) [inline tests/feeds/test_eth_usd_feed.py:5]: Can you add a test for the spot price feeds if they're using different sources?

File: tests/reporters/test_custom_flex_reporter.py [hunk 4]
```diff
@@ -9,7 +9,7 @@
 from telliot_core.utils.response import ResponseStatus
 
 from telliot_feeds.datafeed import DataFeed
-from telliot_feeds.feeds.eth_usd_feed import eth_usd_median_feed
+from telliot_feeds.feeds.eth_usd_legacy_feed import eth_usd_legacy_feed
```
Comment (oraclown) [inline tests/reporters/test_custom_flex_reporter.py:12]: Use spot price feeds here

File: tests/reporters/test_custom_x_reporter.py [hunk 5]
```diff
@@ -19,7 +19,7 @@
 from web3.datastructures import AttributeDict
 
 from telliot_feeds.datafeed import DataFeed
-from telliot_feeds.feeds.eth_usd_feed import eth_usd_median_feed
+from telliot_feeds.feeds.eth_usd_legacy_feed import eth_usd_legacy_feed
```
Comment (oraclown) [inline tests/reporters/test_custom_x_reporter.py:22]: Use spot price feeds

File: tests/reporters/test_interval_reporter.py [hunk 6]
```diff
@@ -15,7 +15,7 @@
 from web3.datastructures import AttributeDict
 
 from telliot_feeds.datafeed import DataFeed
-from telliot_feeds.feeds.eth_usd_feed import eth_usd_median_feed
+from telliot_feeds.feeds.eth_usd_legacy_feed import eth_usd_legacy_feed
```
Comment (oraclown) [inline tests/reporters/test_interval_reporter.py:18]: spot price feeds instead

File: tests/reporters/test_polygon_reporter.py [hunk 7]
```diff
@@ -5,7 +5,7 @@
 from telliot_core.utils.response import ResponseStatus
 
 from telliot_feeds.datafeed import DataFeed
-from telliot_feeds.feeds.eth_usd_feed import eth_usd_median_feed
+from telliot_feeds.feeds.eth_usd_legacy_feed import eth_usd_legacy_feed
```
Comment (oraclown) [inline tests/reporters/test_polygon_reporter.py:8]: Use spot price feeds

File: tests/utils/test_oracle_write.py [hunk 8]
```diff
@@ -4,7 +4,7 @@
 from telliot_core.utils.response import ResponseStatus
 from web3.datastructures import AttributeDict
 
-from telliot_feeds.feeds.eth_usd_feed import eth_usd_median_feed
+from telliot_feeds.feeds.eth_usd_legacy_feed import eth_usd_legacy_feed
```
Comment (oraclown) [inline tests/utils/test_oracle_write.py:7]: Use spot price feeds

**Review by oraclown — CHANGES_REQUESTED** [review]
Thanks @spuddyminer ! Just some small changes needed. The failing code formatting is unrelated and fixed in another PR, so just ignore

**Comment by spuddyminer** [comment]
All tests have been changed as requested.

**Review by oraclown — APPROVED** [review]
Thanks @spuddyminer

**Pull request merged**